src/main/java/com/codefellowship/codefellowship/ApplicationUser.java

```diff
@@ -0,0 +1,38 @@
+package com.codefellowship.codefellowship;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.servlet.view.RedirectView;
+
+public class ApplicationUser {
+
+    //signup page
+
+    @Autowired
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    @ResponseBody
+    @RequestMapping(value = "/signup", method = RequestMethod.GET)

```

Comment:
This isn't a controller, it's an entity; you've missed a lot of the basics of what makes an ApplicationUser distinct from a Controller, or what makes a model different from a controller.